source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/074_DML_PIMS_CLAIM.sql

```diff
@@ -235,4 +235,31 @@ INSERT [dbo].[PIMS_CLAIM]
     ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID])
 VALUES 
     (42, N'04f3a5e1-0423-4489-ab39-96c8f7801c29', N'activity-delete', N'04f3a5e1-0423-4489-ab39-96c8f7801c29', N'Ability to delete activities.', 0, 1, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId, @seedTime, @appUserId, @appUserGuid, @appUserDirectory, @seedTime, @dbUserId)
-GO
\ No newline at end of file
+
+INSERT [dbo].[PIMS_CLAIM]
```

Comment:
Hmm, didn't you say you were going to assign this to Doug?

Reply:
I am adding the Claim, but not the Role_Claims.  Should I just not add the claim either?

Reply:
I would probably either do neither or both.

Reply:
Removed altogether.